templates/crud_rest_api_dartfrog/TEMPLATE.md

```diff
@@ -0,0 +1,92 @@
+---
+name: Simple DartFrog Server
+description: Building a simple CRUD REST API backend with DartFrog
+tags: ["dart", "dartfrog", "globe"]
+username: Invertase
+---
+
+# Simple DartFrog Server
+
+## Overview
+
+Creating a simple CRUD REST API server. This template helps you to save time to scaffold your Dart backend with DartFrog as quickly as possible.
+
+### Getting Started
+
+#### Bootstrap
+
+Initialize your project using the command below
+
+```shell
+$ globe create -t crud_rest_api_dartfrog
+```
+
+#### Start Server
+
+```shell
+$ dart_frog dev --port=3000
+```
+
+### REST Endpoints
+
+- List Repositories
+
+  ```shell
+  curl --request GET --url http://localhost:3000/repos
+  ```
+
+  ```json
+  [
+    { "id": 0, "name": "express", "url": "github.com/expressjs/express" },
+    { "id": 1, "name": "stylus", "url": "github.com/learnboost/stylus" },
+    { "id": 2, "name": "cluster", "url": "github.com/learnboost/cluster" }
+  ]
+  ```
+
+- Create New Repository
+
+  ```shell
+  curl -X POST -H "Content-Type: application/json" -d '{"name": "java", "url": "java.com/env"}' http://localhost:3000/repos\?username\=jane
+  ```
+
+  ```json
+  [
+    { "id": 0, "name": "express", "url": "github.com/expressjs/express" },
+    { "id": 1, "name": "stylus", "url": "github.com/learnboost/stylus" },
+    { "id": 2, "name": "cluster", "url": "github.com/learnboost/cluster" },
+    { "id": 3, "name": "java", "url": "java.com/env" }
```

Comment:
feels very random, could be dart related repos instead

Reply:
Resolved.